#Set the global costmap for navigation with a static map
#Independent settings for the planner's costmap
global_costmap:
    publish_voxel_map: false
    global_frame: map
    robot_base_frame: base_footprint
    update_frequency: 1.0 #Hz
    publish_frequency: 0.0
    static_map: true
    rolling_window: false
